waffle_utils/dataset/dataset.py

```diff
@@ -204,16 +206,16 @@ def load(cls, name: str, root_dir: str = None) -> "Dataset":
     def from_coco(
         cls,
         name: str,
-        coco_file: str,
-        coco_root_dir: str,
```

Comment:
why did it changed?

Reply:
The name `coco` is more like format type, not the image type. So it's confusing what the `coco_root` means. It looks like the `coco_root_dir` represents the dataset root dir, which already exists as `root_dir`. The `coco_root_dir` here is actually the images directory which is independent to the format type. 
```
# mnist sample dataset directories
dataset/  # `root_dir`
    ├ images/  # `coco_root_dir`? 🤔
    │   ├ 1.png
    │   ├ 2.png
    │   └ ...
    └ coco.json  # `coco_file`
```

Reply:
I agree. 
These types of changes should be notified to TX team and need to update our documentation.
Say it loud!